.gitignore

```diff
@@ -12,7 +12,6 @@
 bin/
 log/
 vendor/
-public/
```

Comment:
`public` is used to store static files, I think all of the files in this folder should not be under version control.

Reply:
I agree. The problem was that the `frontend/public` folder was not staged at all, but it is required to be pushed, as it contains the `index.html` and [other resources](https://github.com/free5gc/webconsole/tree/main/frontend/public). I messed up fixing that.